src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final ShoppingCartService shoppingCartService;
+    private final UserService userService;
+    private final DtoResponseMapper<OrderResponseDto, Order> orderDtoResponseMapper;
+
+    public OrderController(OrderService orderService,
+                           ShoppingCartService shoppingCartService,
+                           UserService userService,
+                           DtoResponseMapper<OrderResponseDto, Order> orderDtoResponseMapper) {
+        this.orderService = orderService;
+        this.shoppingCartService = shoppingCartService;
+        this.userService = userService;
+        this.orderDtoResponseMapper = orderDtoResponseMapper;
+    }
+
+    @PostMapping("/complete/{userId}")
+    public OrderResponseDto complete(@PathVariable Long userId) {
+        ShoppingCart shoppingCart = shoppingCartService.getByUser(userService.get(userId));
+        Order order = orderService.completeOrder(shoppingCart);
+        return orderDtoResponseMapper.toDto(order);
+    }
+
+    @GetMapping("/{userId}")
+    public List<OrderResponseDto> get(@PathVariable Long userId) {
+        return orderService.getOrdersHistory(userService.get(userId))
+                .stream()
+                .map(orderDtoResponseMapper::toDto)
+                .collect(Collectors.toList());
+    }
```

Comment:
use request parameter in this endpoins 
<img width="867" alt="image" src="https://user-images.githubusercontent.com/61326568/207250234-770e7dff-a397-45db-ab87-485ceacb9a4e.png">